var/spack/repos/builtin/packages/liggghts/package.py

```diff
@@ -26,7 +26,11 @@ class Liggghts(MakefilePackage):
     variant('profile', default=False,
             description='Generate profiling code')
 
-    depends_on('vtk@6.1.0:8.2.0')
+    variant('libs', default='SHARED',
+            values=('ALL', 'SHARED', 'STATIC', 'NONE'),
```

Comment:
Can we use a multi-valued variant here? Also, we use lowercase letters for variant values in all the other packages

Reply:
Fixed with latest commit